RELEASE.md

```diff
@@ -145,6 +145,16 @@ After a new stable `v$MAJOR.$MINOR` release branch has been created, update the
 `target-branch` and `commit-message.prefix` fields in `.github/dependabot.yml`
 to point to the newly created branch, instead of the old stable branch.
 
+## Update the `renovate` configuration
+
+After a new stable `v$MAJOR.$MINOR` release branch has been created, update `renovate.json5`:
+
+- Add the new branch to the `baseBranches` field and remove the previous release branch (we only maintain the most recent Hubble release).
+- Add the new branch to the `all-go-deps-stable` packageRule in the `matchBaseBranches` field and remove the previous release branch.
+- Update or add a new `golang-images-stable` packageRule to pin the Go version used to the current minor version for the release. (TODO: Add link to example)
```

Comment:
As I understood it it's TODO because currently we can't link to a patch/commit that does it, the idea being that this TODO would be updated the next time we cut a minor release.

Reply:
Yeah, exactly. We don't have to do it when we cut a release, just after the PR is merged, then we can link to the `renovate.json5` on the right line number in master.